### ISSUE TITLE
MS Speed Hack is broken (very detailed data)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MrColdbird/procfw#73]
```
What steps will reproduce the problem?
1. MS Speed hack active
2. Running God of War: Ghost of Sparta in CSO
3. Cutscenes will run choppy.

What is the expected output? What do you see instead?

Expected output: Clear fluid viewing of the cutscenes and loading times reduced 
when in CSO.

Instead: Please see info below

What version of the product are you using? On what operating system?

Please see info below.

Please provide any additional information below.


Results of MS Speed Hack on 
PSP1000 1g and PSP3000 3g

All test were using Nighty "c6c1061605b7"
using God of War Ghost of Sparta in CSO format.

Notable framerate when playing cutscenes (example:
The begining of a new game).


CPU Clocks both at default (222/111hz)
MS Speed Hack set to: Always
ISO Cache is on (by default)
No plugins active
No other tweaks are active

PSP1000 1g
6.20 PRO-B7=Failed.
6.35 PRO-B7=Working but sound is a little choppy, almost unoticeable, but 
graphicly ok.
6.39 PRO-B7=Working but sound is a little choppy, almost unoticeable, but 
graphicly ok.

PSP3000 3g
6.20 PRO-B7=Failed.
6.35 PRO-B7=Working Partially, Still a little choppy graphicly and sound.
6.39 PRO-B7=Working Partially, Still a little choppy graphicly and sound, less 
choppy than 6.35

In conclusion:

MS Speed is fully broken on 6.20 PRO-B7 "c6c1061605b7"
for both 1g and 3g, possibly all other Xg, but I don't
have a way to know.

And for the most part of it, this feature needs either a lot of work, or going 
back to the
original module that doesn't suffer these symptoms.


I also noticed that it works better on my 1g than my 3g.
Now that's funky cause 3g is suppost to be better hardware :S

Current version that works flawlessly on both systems and on all 3 firmwares 
is: "f6312063516a"


```

Original issue reported on code.google.com by `alsalo1...@gmail.com` on 16 Jun 2011 at 1:38
